packages/blog/src/App.js

```diff
@@ -40,9 +47,17 @@ function App() {
                             <BlogPostTeaser/>
                             </div>
                         </Route>
+
+                        {/*<Route path="/blogpostxy">*/}
+                        {/*    <BlogDetailView/>*/}
+                        {/*</Route>*/}
+
                         <Route path="/">
                             <Home/>
                         </Route>
+
+                        <Route exact path="/blogpost/:id" render={(props) => <BlogDetailView {...props} />} />
```

Comment:
Huch, die Variante kannte ich auch noch nicht! :)
Aber mache es doch bitte, wie in Zeile 56 und pack den <BlogDetailView /> Tag einfach als children in die Route. Dann ist es konsistent und "so wie man es erwarten würde.